### DISCUSSION TITLE
Allow for Percurro Shamir/Teleport while in PvP

### DISCUSSION THREAD

**Discussion by aisukuma** [Stardust-Labs-MC/StardustMC#35] [Ideas]
As per #18, what do you think of enabling the option to teleport during PvP? 

I don't think I can determine if a player can teleport away vs using the Percurro Shamir.

**Comment by CodingPanda0623** [comment]
I believe it should be enabled because it would make pvp a lot more interesting, though there should be some sort of punishment if someone tries to teleport to get out (they lose the match, starz taken away, etc.)
